ITKIOScanco_UnitConversion.ipynb

```diff
@@ -0,0 +1,227 @@
+{
```

Comment:
<p>For now I've implemented this using the example data referred to <a href="https://github.com/KitwareMedical/ITKIOScanco/blob/master/examples/ConvertScancoVolumesToOpenStandardFormats.ipynb" target="_blank">here</a>. I'm not entirely sure how this is supposed to function in a general sense, since the URL provided in the example may not be the location where these images will be found. The hash value seems to be an important argument for pooch.retrieve as well, but it's not clear to me how this will be obtained.</p>